employee.html

```diff
@@ -0,0 +1,215 @@
+                </div>
+                <div class="salary-expenses row">
+                    <div class="col-6">
+                        <div class="salary-col text-center">
+                            <span class="small-tag">Salary</span>
+                            <div class="salary-block remuneration">
+                                <!--${{ employee.remuneration|intcomma }} -->
+                                <div id="tab1" class="tabcontent">100</div>
+                                <div id="tab2" class="tabcontent">200</div>
+                                <div id="tab3" class="tabcontent">300</div>
+                            </div>
+
+                            <div class="salary-rank">{{ rank.overall.salary|ordinal }} overall</div>
+                            {% if rank.faculty.salary %}
+                            <div class="salary-rank">{{ rank.faculty.salary|ordinal }} in {{ employee.faculty.short_name }}</div>
+                            {% endif %}
+                        </div>
+                    </div>
+                    <div class="col-6">
+                        <div class="salary-col text-center">
+                            <span class="small-tag">Expenses</span>
+                            <div class="salary-block expenses">
+                                ${{ employee.expenses|intcomma }}
+                            </div>
+                            <div class="salary-rank">{{ rank.overall.expenses|ordinal }} overall</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-8">
+                <div class="charts">
+                    <div class="chart">
+                    </div>
+                    {% if employee.faculty %}
+                        <ul class="chart-tabs">
+                            <li><a href="#chart-overall">Overall</a></li>
+                            <li><a href="#chart-faculty">{{ employee.faculty.full_name }}</a></li>
+                        </ul>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block scripts %}
+<script>
+    var EMPLOYEE = {{ employee.remuneration }};
+    var ID = {{ employee.id }};
+    {% if employee.faculty %}
+        var FACULTY = {{ employee.faculty.id }};
+    {% else %}
+        var FACULTY = false;
+    {% endif %}
+</script>
+<script src="{% static "js/fb.js" %}"></script>
+<script src="{% static "js/d3.min.js" %}"></script>
+<script src="{% static "js/employee.js" %}"></script>
+
+<script>
+document.getElementById("defaultOpen").click();
+function openSalary(evt, salary) {
+    // Declare all variables
+    var i, tabcontent, tablinks;
+
+    // Get all elements with class="tabcontent" and hide them
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+
+        tabcontent[i].style.display = "none";
```

Comment:
jQuery is included on the salary site (it is included in the base template [here](https://github.com/ubyssey/salarydb/blob/master/templates/base.html#L42); you can confirm by opening the console on the page and evaluating `$`), so you can dramatically simplify these loops:

```js
$('.tabcontent').css('display', 'none');

$('.tablinks').removeClass('active');

$('#' + salary).css('display', 'block');

$(evt.currentTarget).addClass('active');
```